python/dolfinx/wrappers/geometry.cpp

```diff
@@ -241,7 +241,7 @@ void geometry(py::module& m)
           [](const dolfinx::geometry::BoundingBoxTree& self,
              const std::size_t i)
           {
```

Comment:
This function is more complicated than it needs to be. Just create a `py::array_t` from `bbox` pointer and size (6). By default pybind11 will copy the data.